app/src/components/AccountTab/AccountTab.tsx

```diff
@@ -45,23 +68,29 @@ const AccountTab = ({
                     value={modals.remove}
                     closeModal={toggleModal("remove")}
                     handleRemove={handleRemove}
-                    display={removeAccountDisplay(modals.remove)}
+                    display={removeAccountDisplay("nope")}
```

Comment:
is this "nope" standard ts?

Reply:
Nope. The modal message maker that is there expected a string and that to be a string describing an enode. After switching it to typescript I had to fix the type error but the modal is completely wrong no matter what gets passed into it. It’s getting fixed properly during the add or the delete, one of the upcoming PRs.